test-harness/infratests/unit.go

```diff
@@ -1,30 +1,28 @@
 /*
-This file provides abstractions that simplify the process of unit-testing terraform templates. The goal
+Package infratests This file provides abstractions that simplify the process of unit-testing terraform templates. The goal
```

Comment:
"Package infratests" - Why is this phrase positioned here?

Reply:
This is recommended based on the `go-lint` plugin I have installed in VS Code.